### PR TITLE
chore(schema): drop the unused geocode_cache table

### PR DESCRIPTION
**Pull request by jdalsem**

### CODE REVIEW

File: docs/guides/hooks-list.rst [hunk 1]
```diff
@@ -30,9 +30,6 @@ System hooks
 **validate, input**
 	Filter GET and POST input. This is used by ``get_input()`` to sanitize user input.
 
-**geocode, location**
```
Comment (hypeJunction) [inline docs/guides/hooks-list.rst]: Is the hook not triggered anywhere anymore?

**Review by hypeJunction — COMMENTED** [review]
This deserves a mention in the upgrading docs.

File: docs/guides/hooks-list.rst (hunk 1 above)
Comment (jdalsem) [inline docs/guides/hooks-list.rst]: nope

**Comment by jdalsem** [comment]
added mention in docs

**Pull request merged**